configs/_base_/models/fcn_unet_s5-d16.py

```diff
@@ -52,6 +53,7 @@
         num_classes=2,
         norm_cfg=norm_cfg,
         align_corners=False,
+        ignore_index = 255,
```

Comment:
Same reason above, this line is probably unnecessary.

Reply:
Thanks for reviewing! I have deleted this code.